src/jobs/Recruit.Vacancies.Jobs/TrainingTypes/Models/ApprenticeshipType.cs

```diff
@@ -0,0 +1,8 @@
+namespace Esfa.Recruit.Vacancies.Jobs.TrainingTypes.Models
+{
+    public enum ApprenticeshipType
```

Comment:
In v1 the 'ApprenticeshipType' Enum is used for Level. E.g. Intermediate, Advanced, Higher, Foundation Degree, Degree, Masters.  Wondering if we should call this Enum something else to avoid any possible confusion? TrainingType maybe?

Reply:
Sorry didn't read your comment before merging in. Will make the change to avoid confusion.